README.md

```diff
@@ -19,13 +19,11 @@ This library is a new design that inherits from both the AWS IoT Device SDK Embe
 - Configurable memory allocation (static-only or dynamic). Memory allocation functions may also be set by the user.
 - Network stack based on OpenSSL when building for Linux.
 - MQTT persistent session support.
-- Device Defender library.
+- Device Defender client library for the [AWS IoT Device Defender](https://docs.aws.amazon.com/iot/latest/developerguide/device-defender.html) service.
+- Provisioning client library for the [Fleet Provisioning  feature
+of AWS IoT Core](https://docs.aws.amazon.com/iot/latest/developerguide/provision-wo-cert.html).
 
-Compared to the AWS IoT Device SDK Embedded C v3.0.1, the following features were removed:
-- Auto-reconnect for MQTT connections.
-- Shadow JSON document generator.
-
-## Building and Running Demos
+## Building and Running the Fleet Provisioning library Demo
```

Comment:
Is this section specific to Fleet provisioning demo? What about other demos?

Reply:
Oops, it is an artifact of the `FleetProvisioning_beta` README. Will remove the reference to the library from the section